config/seata-config-core/src/main/resources/registry.conf

```diff
@@ -26,6 +26,15 @@ registry {
     cluster = "default"
     serverAddr = "127.0.0.1:8500"
   }
+  sofa {
+    serverAddr = "127.0.0.1:9603"
+    application = "default"
+    region = "DEFAULT_ZONE"
+    datacenter = "DefaultDataCenter"
+    cluster = "default"
+    group = "SEATA"
```

Comment:
SEATA_GROUP?

Reply:
yes， It is a label for sofa registry server， it will store this service in group "SEATA"，

Reply:
> yes， It is a label for sofa registry server， it will store this service in group "SEATA"，

Ok， I will change it